windows/src/engine/keymanhp/keymanx64.rc

```diff
@@ -2,5 +2,5 @@
 //
 #include "versionx64.rc"
 #include "manifest.rc"
-1 ICON  "..\\..\\desktop\\branding\\keymandesktop90-64.ico"
```

Comment:
Do we want to delete the keymandesktop90-64.ico file? Or is it used elsewhere?

Reply:
Good point I will check.